data/beps/beps.yml

```
@@ -279,16 +276,17 @@
     content:
     -   raw
     leads:
-    -   name: Sylvain Takerkart
-        email: sylvain.takerkart@univ-amu.fr
-    -   name: Julia Sprenger
-        email: julia.sprenger@univ-amu.fr
+    -   name: Yaroslav O. Halchenko
+        email: debian@onerussian.com
+    -   name: Ben Dichter
+        email: ben.dichter@gmail.com
```

yes, somewhat.  AFAIK @SylvainTakerkart is still involved/leading but @bendichter is taking over Julia.  I am participant, not marked as a lead, so better be removed (can't suggest on deleted lines I believe)

```suggestion
    -   name: Sylvain Takerkart
        email: sylvain.takerkart@univ-amu.fr
    -   name: Ben Dichter
        email: ben.dichter@gmail.com
```